docs/source/x_ext.rst

```diff
@@ -150,7 +151,7 @@ This non-compressed instruction is then attempted for offload via the issue inte
 Offloading of the (non-compressed, 32-bit) instructions happens via the issue interface. 
 The external |coprocessor| can decide to accept or reject the instruction offload. In case of acceptation the |coprocessor|
 will further handle the instruction. In case of rejection the core will raise an illegal instruction exception. 
-As part of the issue interface transaction the core provides the instruction and required register file operand(s) to the |coprocessor|. If
+Subsequently, the core provides the required register file operand(s) to the |coprocessor| via the register interface. If
```

Comment:
I don't like the word 'Subsequently' here. It should be something like 'Coincidentally or subsequentially'

Reply:
what do you not like about this word? subsequentially does not seem to be an English word.

Reply:
ok will reword.

Reply:
fixed